src/50_Samples_%26_Templates/Product.html

```diff
@@ -475,17 +477,16 @@ <h2 class="m1">Apple</h2>
       depending on the selected colour.
 
     We bind the `text` attribute to the
-      value of expression `product[selectedColor].price`. `product` is the id `amp-state` json described in the Product Configuration section.
+      value of expression `product[product.selectedColor].sizes[product.selectedSize]`. `product` is the id `amp-state` json described in the Product Configuration section.
 
-    Notice how we set a default value for the price: values defined inside `amp-state`
-      will not be accessed at the page load. We set `green` and `S` as default color and default size
-      as the correspondent variables have `null` value at default.
+    Notice how we set $1.99 as default value for the price:  expressions
```

Comment:
1.99 is the default value for the price before the expression is evaluated -> the value that will be shown at load time.
I can remove "value" and let it be "Notice how we set $1.99 as default for the price", would it make clearer?